node/src/Commands.ts

```diff
@@ -1961,6 +1973,124 @@ export function createGeoHash(
     return createCommand(RequestType.GeoHash, args);
 }
 
+// This type unites `GeoSearchResultOptions` and `GeoSearchOptions` of the java client
```

Comment:
Do we need this comment?

Reply:
Only for us, a user doesn't see that

Reply:
let's remove this - we don't need to mention the Java client